index.bs

```diff
@@ -2634,8 +2637,17 @@ Multiple [=constructor operations=] may appear on a given [=interface=].
 For each [=constructor operation=] on the [=interface=], there will be a way to attempt to
 construct an instance by passing the specified arguments.
 
-The prose definition of a [=constructor operation=] must either initialize the value passed as
-<b>[=this=]</b>, or throw an exception.
+The constructor description of a [=constructor operation=] must either initialize the value passed
+as [=this=], or throw an exception.
+
+The <dfn>constructor steps</dfn> of a constructor _constructor_ should be introduced using text of
```

Comment:
`_constructor_` should be `|constructor|`, no? Also do we want to say `[=constructor operation=] |constructor|`? It seems moderately important to distinguish between constructor operations and constructors since the latter link to [JavaScript's definition](https://tc39.github.io/ecma262/#constructor) in the document.

Reply:
Hopefully fixed.